tools/make/diplomat-coverage/src/allowlist.rs

```diff
@@ -344,6 +344,9 @@ lazy_static::lazy_static! {
         "icu::calendar::Date::wrap_calendar_in_arc",
         "icu::calendar::Date::wrap_calendar_in_ref",
 
+        // Utility for use by other ICU4X crates
```

Comment:
not sure this is the best justification. the main reason why we cannot expose it is because it's a generic collection. we *could* expose `PluralElements<str>` or something over FFI. we don't because it's mainly used for data struct internals

Reply:
Right, it's a utility for use by data struct internals. I'll clarify.